app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = I18n.t("title")
+    base_title if page_title.empty?
```

Comment:
ko return thì nó sẽ thực hiện tiếp line, cuối cùng thì kết quả sẽ là xử lý line 5

==> Có thể dùng toán tử 3 ngôi hoặc if else
page_title.empty? ? base_title : "#{page_title}| #{base_title}"

Reply:
dạ vâng chị